.github/Issue_Template/Feature.yml

```
@@ -0,0 +1,26 @@
+name: 💡Feature 
+title: '[FEATURE] <description>'
+labels: ['⭐ goal: addition','triage/accepted','triage/awaiting']
```

```suggestion
labels: ['triage/awaiting', 'enhancement']
```
We don't have `goal: addition` label. Maintainers replace `triage/awaiting` with `triage/accepted` after going through the issue. I propose GitHub's default `enhancement` label for Feature requests.